website/docs/r/storage_account.html.markdown

```diff
@@ -229,6 +233,22 @@ A `queue_properties` block supports the following:
 
 * `hour_metrics` - (Optional) A `hour_metrics` block as defined below.
 
+---
+
+A `blob_account_properties` block supports the following:
+
+* `static_website` - (Optional) A `static_website` block as defined below.
+
+---
+
+A `static_website` block supports the following:
```

Comment:
Should the documentation mention that a $web container is created and the files should be placed inside?

Reply:
Good call, added!